ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetTransactionProcessor.java

```diff
@@ -500,7 +500,8 @@ public TransactionProcessingResult processTransaction(
       LOG.error("Critical Exception Processing Transaction", re);
       return TransactionProcessingResult.invalid(
           ValidationResult.invalid(
-              TransactionInvalidReason.INTERNAL_ERROR, "Internal Error in Besu - " + re));
+              TransactionInvalidReason.INTERNAL_ERROR,
+              "Internal Error in Besu - " + re + "\n" + printableStackTraceFromThrowable(re)));
```

Comment:
what does this look like in reality?

Reply:
This is an example:

`Internal Error in Besu - java.lang.IllegalStateException
    at com.google.common.base.Preconditions.checkState(Preconditions.java:486)
    at net.consensys.linea.zktracer.module.hub.stack.Stack.processInstruction(Stack.java:333)
    at net.consensys.linea.zktracer.module.hub.Hub.handleStack(Hub.java:376)
    at net.consensys.linea.zktracer.module.hub.Hub.processStateExec(Hub.java:470)
    at net.consensys.linea.zktracer.module.hub.Hub.tracePreOpcode(Hub.java:705)
    at net.consensys.linea.zktracer.ZkTracer.tracePreExecution(ZkTracer.java:101)
    at org.hyperledger.besu.evm.EVM.runToHalt(EVM.java:187)
    at org.hyperledger.besu.evm.processor.AbstractMessageProcessor.codeExecute(AbstractMessageProcessor.java:174)
    at org.hyperledger.besu.evm.processor.AbstractMessageProcessor.process(AbstractMessageProcessor.java:196)
    at org.hyperledger.besu.ethereum.mainnet.MainnetTransactionProcessor.process(MainnetTransactionProcessor.java:511)
    at org.hyperledger.besu.ethereum.mainnet.MainnetTransactionProcessor.processTransaction(MainnetTransactionProcessor.java:418)
    at org.hyperledger.besu.ethereum.mainnet.MainnetTransactionProcessor.processTransaction(MainnetTransactionProcessor.java:206)
    at org.hyperledger.besu.services.TraceServiceImpl.lambda$trace$6(TraceServiceImpl.java:217)
    at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
    at org.hyperledger.besu.services.TraceServiceImpl.trace(TraceServiceImpl.java:202)
    at org.hyperledger.besu.services.TraceServiceImpl.lambda$trace$4(TraceServiceImpl.java:182)
    at org.hyperledger.besu.ethereum.api.jsonrpc.internal.processor.Tracer.lambda$processTracing$0(Tracer.java:52)
    at org.hyperledger.besu.ethereum.api.query.BlockchainQueries.lambda$getAndMapWorldState$23(BlockchainQueries.java:923)
    at java.base/java.util.Optional.flatMap(Optional.java:289)
    at org.hyperledger.besu.ethereum.api.query.BlockchainQueries.getAndMapWorldState(BlockchainQueries.java:919)
    at `